crates/rpc/rpc/src/eth/api/mod.rs

```diff
@@ -50,6 +51,12 @@ mod transactions;
 use crate::BlockingTaskPool;
 pub use transactions::{EthTransactions, TransactionSource};
 
+/// Type alias for a watch receiver that receives the recent pending block with receipts
+pub(crate) type LocalPendingBlockWatcherReceiver =
+    watch::Receiver<Option<(SealedBlock, Vec<Receipt>)>>;
```

Comment:
we want a wrapper type for this, this makes it easier to make additional changes to the types we're sending and is nicer